api/overview/azure/latest/microsoft.azure.servicebus-readme.md

```diff
@@ -5,10 +5,9 @@ author: ramya-rao-a
 ms.author: ramyar
 ms.date: 11/08/2021
 ms.topic: reference
-ms.prod: azure
 ms.technology: azure
 ms.devlang: dotnet
-ms.service: servicebus
+ms.service: service-bus
```

Comment:
This should be updated in https://github.com/Azure/azure-sdk/blob/main/_data/releases/latest/dotnet-packages.csv